src/components/panels/EventsPanel.vue

```diff
@@ -88,6 +99,24 @@ export default {
       }
       return labelClass;
     },
+    eventHealthVariantOpposite(event) {
+      const health = event.health;
+      let labelClass = 'success';
+      if (health >= 20 && health < 50) {
+        labelClass = 'info';
+      } else if (health >= 50 && health < 80) {
+        labelClass = 'warning';
+      } else if (health >= 80) {
+        labelClass = 'danger';
+      }
+      return labelClass;
+    },
+    isHealthReversed(event) {
+      if (event.description == 'Thermia Fractures') {
```

Comment:
let's make this a global list in the component and do an `array.includes` on it to check